### ISSUE TITLE
Softmax API Mismatch on beta param between TF and TFL 

### ISSUE THREAD

**Issue by Jerry-Ge** [tensorflow/tensorflow#60435]
In TFL's reference for Softmax, a beta param is supported to be multiplied with logits. [[Link](https://github.com/tensorflow/tensorflow/blob/fddb46a1b7f0136f64442629108961560b5e3486/tensorflow/lite/kernels/internal/reference/softmax.h#L53)] 

However, in TF, the beta seems has not been supported yet [[Link](https://www.tensorflow.org/api_docs/python/tf/nn/softmax)]. Is there any reason why it's only being supported in TFL not in TF now? 

More background: I'm trying to test the beta param from Softmax with a model converted from TF to TFL. So if it doesn't support beta in TF, how could I insert beta into the TFL model? 

Thanks, 

Jerry

**Comment by sachinprasadhs** [comment]
Hi, The original PR https://github.com/tensorflow/tensorflow/pull/42330 with the beta implementation in TFL has the detailed explanation, let me know if that helps you in understanding

**Comment by Jerry-Ge** [comment]
> Hi, The original PR #42330 with the beta implementation in TFL has the detailed explanation, let me know if that helps you in understanding

Thanks Sachin for the note. But I haven't found the explanation of why TF is supporting the beta parameter while TFL is not? That PR seems is only explaining the need to remove some duplicated calculations for exp. 

Jerry

**Comment by pjpratik** [comment]
Hi @Jerry-Ge 

The PR #60541 has been merged with the commit https://github.com/tensorflow/tensorflow/commit/ecbe847f8f3841898ea6dfa482b99a3bca7f4e6f with following lines

https://github.com/tensorflow/tensorflow/blob/70c60acb6100651def0b8447783ecb65e273f155/tensorflow/compiler/mlir/tosa/transforms/legalize_common.cc#L1493-L1499

Could please confirm and close this issue if it is resolved?

Thanks.
